data/src/main/scala/ch.epfl.scala.index.data/Main.scala

```diff
@@ -0,0 +1,132 @@
+package ch.epfl.scala.index.data
+
+import java.nio.file.Path
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import ch.epfl.scala.index.data.bintray.{BintrayDownloadPoms, BintrayListPoms}
+import ch.epfl.scala.index.data.cleanup.NonStandardLib
+import ch.epfl.scala.index.data.elastic.SeedElasticSearch
+import ch.epfl.scala.index.data.github.GithubDownload
+import ch.epfl.scala.index.data.maven.DownloadParentPoms
+import org.slf4j.LoggerFactory
+
+import scala.sys.process.Process
+
+/**
+  * This application manages indexed POMs.
+  */
+object Main {
```

Comment:
I basically moved the `BintrayPipeline` into a `Main`, to make it clearer that it’s the app entry point.